### ISSUE TITLE
Provision of 19115 metadata for images and gridded data

### ISSUE THREAD

**Issue by juliandh81** [ISO-TC211/19115-2Revision#27]
I have a generic doubt about the provision of 19115-1 and 19115-2 metadata for raster data. Should data providers choose between 19115-1 and 19115-2 metadata, or provide both? In other words:
-   Raster data can be accompanied by 19115-1 ‘OR’ 19115-2 metadata
-   Raster data can be accompanied by 19115-1 ‘AND/OR’ 19115-2 metadata

The clarification of this issue can help me in the revision.

Best

**Comment by SJRowlison** [comment]
My understanding is that 19115-2 is just an extension of 19115-1. Raster data may be accompanied by 19115 compliant metadata which contains both 19115-1 and 19115-2 information.

**Comment by DaveDanko** [comment]
agree, 19115-1 AND 19115-2. -2 will allow you to provide additional source and processing/algorithm information, acquisition metadata, etc.
